x-pack/plugins/security_solution/public/common/store/sourcerer/model.ts

```diff
@@ -27,7 +27,7 @@ export interface ManageScope {
   docValueFields: DocValueFields[];
   errorMessage: string | null;
   id: SourcererScopeName;
-  indexPattern: IIndexPattern;
+  indexPattern: Omit<IIndexPattern, 'fieldFormatMap'>;
```

Comment:
this attribute `fieldFormatMap` was giving us some issue but since we are not using it through the app, i removed it for now. We also have a PR in the pipeline to remove the type` IIndexPattern` to use `DataViewSpec`

https://github.com/elastic/kibana/pull/111461